website/src/content/development/notifications.mdx

```diff
@@ -0,0 +1,343 @@
+---
+title: Notifications
+---
+
+In the previous sections, we learned how to develop forms. In this section we focus on providing users with relevant information in response to a successful or failed form submission.
+
+To facilitate the use of notifications, Custom Applications come with a built-in [notification system](https://docs.commercetools.com/custom-applications/api-reference/commercetools-frontend-actions-global#notification-system).
+
+# Notification system
+
+Notifications are an important way of providing feedback in response to user actions.
+Submission of a form might end with a success, but for various reasons it might also fail.
+Either way, a user should be informed what was the end result of the action.
+
+The Custom Application built-in notification system provides a way of showing notifications by dispatching Redux actions throught bound action creators or React hooks exposed from the [`@commercetools-frontend/actions-global`](/api-reference/commercetools-frontend-actions-global) package.
+Notifications can be dispatched in one of the predefined placement options (referred to as **domains**, for instance `page` or `side`) and in one of various kinds depending on the domain (for instance `error`, `warning` or `success`).
+
+<Info>
+
+For defining the domain and the type of a notification we recommend using the values exposed from `@commercetools-frontend/constants`.
+
+</Info>
+
+## Side notifications
+
+![Side notifications](/images/notifications-side.png)
+
+## Page notifications
+
+![Page notifications](/images/notifications-api-error.png)
+
+# Notifications best practices
```

Comment:
I would split and include these into each section "Side notifications" and "Page notifications"

Reply:
I would also explain a bit more that side notifications are meant to disappear after a couple of seconds but page notifications do not, and the user has to manually close them.